taipy/gui/viselements.json

```diff
@@ -1227,6 +1227,12 @@
                         "default_value": "False",
                         "doc": "If set to True, multiple files can be uploaded."
                     },
+                    {
+                        "name": "webkitdirectory",
```

Comment:
bad name ... who cares about webkit in python ?

Reply:
I would suggest to get the word `nested`, like `nested_folder_upload` in there, as it captures what the code does. It not only uploads the content of folder, but imitates the structure of the folder in the directory it is uploaded to, like:
| dir1
    | -- file1
    | -- dir2
         |-- file1
         |-- file2

Reply:
that's a nice added benefit but I think the main feature is to be able to select a folder and everything it contains, no ?

Reply:
Not for me, because I need the structure to be preserved for a project, but I can see, why this is mostly the usecase.

Reply:
> bad name ... who cares about webkit in python ? `allow_folder_selection` that would translate to `allowFolderSelection` ? What does our naming expert @FabienLelaquais says ?

What about `files_only`, defaults to True?

Reply:
That would solve it the other way round. In my eyes, it sounds a bit like `files_only=false` would allow the selection of files and folders (which is not the case). Im currently working with `selectFolder = false`, but I don't have a strong preference.

Reply:
I see your point, it is either files of folders ...
Another name @FabienLelaquais ?

Reply:
I'm not sure I understand the question, @FredLL-Avaiga.
To me, `files_only` is pretty clear.
If True (which would be the default), one can select only files.
If False, one can select files and directories.

Reply:
it is not really that way
If one specifies directory to the files input, only directory can be selected

Reply:
Now I get it, after I have read the doc for <input>... sorry.
@FredLL-Avaiga 
What about a new property called `selection_type`. Can be set to "file" or "directory" (or "f"|"d"), defaults to "f"?

Reply:
I think this would be a clear indication of the function. Should I change it to match this naming scheme. I would opt for `selection_type`, as maybe `type` is too broad an could be mistaken for the `input` attribute. It would also would also give room to expand functionality. By defaulting to file it should not interfere with previous code.